pyproject.toml

```diff
@@ -11,7 +11,7 @@ packages = [{ include="scripts", from="." }]
 
 [tool.poetry.dependencies]
 python = ">=3.6, <3.8"
-dcicutils = "<=1.2.1"
+dcicutils = ">=1.2.1"
```

Comment:
I'd suggest ^1.2.1 instead, so that it can be updated until there's a major version change. If 2.0.0 introduces incompatible changes then we don't want to update the version automatically without reviewing the repo, probably.

Reply:
Good suggestion - agreed!